src/plugins/data/common/search/session/types.ts

```diff
@@ -17,82 +17,19 @@
  * under the License.
  */
 
-import { Observable } from 'rxjs';
-import type { SavedObject, SavedObjectsFindResponse } from 'kibana/server';
-
-export interface ISessionService {
```

Comment:
Discussed with @lukasolson, server doesn't use this as interfaces differ. Moved to the client to avoid further confusions.